Automattic/jetpack
# Forms: Dashboard finetuning 2

## Proposed changes:

- [x] “View responses” on mobile: Make cursor pointer and styling adjustments
- [x] Source Column: Show "source path" if the post/page has no title
- [x] Response details panel: Show source full path
- [x] Mobile screens: Should have 24px padding left and right
- [x] Mobile screens: Hide filter dropdowns (showing search only)


### Other information:


## Jetpack product discussion
N/A

## Does this pull request change what data or activity we track or use?
No

## Testing instructions:

Enable the jetpack forms package and the new dashboard before testing with:

```
add_filter( 'jetpack_contact_form_use_package', '__return_true' );
add_filter( 'jetpack_forms_dashboard_enable', '__return_true' );
```

- Go to `/wp-admin/admin.php?page=jetpack-forms`
- Check if the tasks listed above were completed correctly
